[Big Sur] Fix Big Sur build after 251860@main

litherum opened the pull request:
#### f6a54db4f9b1443f6e9166eb8a65632b2ea6d152
<pre>
[Big Sur] Fix Big Sur build after 251860@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=242095">https://bugs.webkit.org/show_bug.cgi?id=242095</a>

Unreviewed build fix.

AXValueState doesn&apos;t exist on Big Sur.

* Source/WTF/wtf/PlatformHave.h:
* Source/WebKit/Platform/spi/Cocoa/AccessibilitySupportSPI.h:

Canonical link: <a href="https://commits.webkit.org/251943@main">https://commits.webkit.org/251943@main</a>
</pre>

aproskuryakov commented on Source/WTF/wtf/PlatformHave.h:
```diff
@@ -1018,6 +1018,14 @@
 #define HAVE_PER_APP_ACCESSIBILITY_PREFERENCES 1
 #endif
 
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000) \
```
We only need the macOS check here, nothing else (including macCatalyst on old macOS) is supported.

litherum committed:
[Big Sur] Fix Big Sur build after 251860@main
https://bugs.webkit.org/show_bug.cgi?id=242095

Unreviewed build fix.

AXValueState doesn't exist on Big Sur.

* Source/WTF/wtf/PlatformHave.h:
* Source/WebKit/Platform/spi/Cocoa/AccessibilitySupportSPI.h:

Canonical link: https://commits.webkit.org/251943@main
